integral_view/views/common.py

```diff
@@ -24,16 +24,27 @@ def view_system_info(request):
             elif request.GET["ack"] == "system_date_set":
                 return_dict['ack_message'] = "Date successfully updated"
             elif request.GET["ack"] == "system_datetime_set":
-                return_dict['ack_message'] = "Date and Time successfully updated"
+                return_dict['ack_message'] = "Date and time successfully updated"
+            elif request.GET["ack"] == 'system_timezone_set':
+                return_dict['ack_message'] = "Timezone successfully updated"
+            elif request.GET['ack'] == 'system_date_timezone_set':
+                return_dict['ack_message'] = 'Date and timezone successfully updated'
+            elif request.GET['ack'] == 'system_time_timezone_set':
+                return_dict['ack_message'] = 'Time and timezone successfully updated'
+            elif request.GET['ack'] == 'system_datetimetz_set':
+                return_dict['ack_message'] = 'Date, time and timezone successfully updated'
 
         si, err = system_info.load_system_config()
         if err:
             raise Exception(err)
         now = datetime.datetime.now()
         milliseconds = int(time.mktime(time.localtime()) * 1000)
+        cmd = "timedatectl | grep 'Time' | grep -oE '(\S+/.+$)'"
```

Comment:
Can you add this also to the integralstor_utils file with a get_timezone() function? That way, it can be used from elsewhere if needed.